Userland/DevTools/HackStudio/HackStudioWidget.cpp

```diff
@@ -1078,15 +1078,21 @@ void HackStudioWidget::build()
     auto result = m_project_builder->build(active_file());
     if (result.is_error()) {
         GUI::MessageBox::show(window(), String::formatted("{}", result.error()), "Build failed", GUI::MessageBox::Type::Error);
-    }
+        m_build_action->set_enabled(true);
+        m_stop_action->set_enabled(false);
+    } else
+        m_stop_action->set_enabled(true);
```

Comment:
If the `if` has braces, then please also add them to the `else`. It's easier to read that way.